text/0224-experimental-attrribute-registration.md

```diff
@@ -0,0 +1,117 @@
+# Experimental Semantic Convention Attribute Registration
+
+A process for the registration of semantic convention attributes not yet included in a semantic convention.
+
+## Motivation
+
+Today, semantic conventions are "all or nothing."
+Each instrumentation implements one or more full semantic conventions with little room for experimentation.
+The lack of experimentation means there is often limited experience and evidence to guide  convention authors.
+It also often leaves instrumentation authors with little or no guidance if they wish to capture telemetry not yet covered by semantic conventions.
+This [Slack thread on #otel-specification](https://cloud-native.slack.com/archives/C01N7PP1THC/p1667997213067169?thread_ts=1667933561.313819&cid=C01N7PP1THC) highlights the problem in context of a specific semantic convention pull request.
+What should OpenTelemetry do with semantic convention proposals when there's unclear evidence that a particular signal is necessary or ubiquitous?
+
+## Explanation
+
+This proposal generally follows the recommendations of [RFC6648 section 4 “Recommendations for Protocol Designers.”](https://www.rfc-editor.org/rfc/rfc6648#section-4)
+RFC6648 recommends a registry of names be established which standardizes usage and prevents collisions.
+This OTEP separates a dictionary of semantic convention attributes from the semantic conventions, to be referenced by semantic conventions when necessary, and establishes a procedure for registering new and experimental semantic convention attributes.
```

Comment:
Just for my understanding: the idea is, that it's easier to add attributes to the dictionary than to add them to the semantic conventions itself. A user can then add an attribute to the dictionary easily, and then experiment without fearing collisions, and eventually write semantic conventions referring the attributes in the dictionary?

Reply:
Exactly. That also means that adding new attributes does not require discussion on where and how the attribute is intended to be used.